actpool/actpool_test.go

```diff
@@ -959,19 +959,22 @@ func TestActPool_GetUnconfirmedActs(t *testing.T) {
 	require.NoError(err)
 	tsf4, err := testutil.SignedTransfer(addr1, priKey1, uint64(4), big.NewInt(30), []byte{}, uint64(100000), big.NewInt(0))
 	require.NoError(err)
-
+	tsf5, err := testutil.SignedTransfer(addr1, priKey2, uint64(1), big.NewInt(30), []byte{}, uint64(100000), big.NewInt(0))
+	require.NoError(err)
 	err = ap.Add(tsf1)
 	require.NoError(err)
 	err = ap.Add(tsf3)
 	require.NoError(err)
 	err = ap.Add(tsf4)
 	require.NoError(err)
+	err = ap.Add(tsf5)
+	require.NoError(err)
 
-	acts := ap.GetUnconfirmedActs(addr2)
-	require.Equal([]action.SealedEnvelope{}, acts)
+	acts := ap.GetUnconfirmedActs(addr3)
+	require.Equal([]action.SealedEnvelope(nil), acts)
```

Comment:
Why nil?

[]action.SealedEnvelope{} doesn't work?

Reply:
[]action.SealedEnvelope{} doesn't work,I think nil is different from empty